src/main/java/com/betting/ground/admin/domain/Report.java

```diff
@@ -0,0 +1,29 @@
+package com.betting.ground.admin.domain;
+
+import com.betting.ground.auction.domain.Auction;
+import com.betting.ground.user.domain.User;
+import jakarta.persistence.*;
+import lombok.*;
+
+@Entity
+@Getter
+@AllArgsConstructor
+@Builder
+@NoArgsConstructor
```

Comment:
```suggestion
@NoArgsConstructor(access = AccessLevel.PROTECTED)
```
의 형태로 접근권한 변경하면 어떨까요

Reply:
수정했습니다